src/Tribe/PUE/Checker.php

```diff
@@ -644,7 +635,7 @@ public function request_update() {
 			//admin display for if the update check reveals that there is a new version but the API key isn't valid.
 			if ( isset( $pluginInfo->api_invalid ) ) { //we have json_error returned let's display a message
 				$this->json_error = $pluginInfo;
-				add_action( 'admin_notices', array( &$this, 'display_json_error' ) );
+				add_action( 'admin_notices', array( &$this, 'display_license_error_message' ) );
```

Comment:
I realize you didn't introduce it, but since we're touching this line anyway can we drop the reference operator (ie `change `&$this` to `$this`)?